assemblies/karaf/pom.xml

```diff
@@ -63,7 +50,7 @@
   <build>
     <resources>
       <resource>
-        <directory>${project.basedir}/src/main/filtered-resources</directory>
```

Comment:
We do not use ${project.basedir}/src/main/filetered-resources folder, we use ${project.basedir}/src/main/resources and the files inside should be filtered